node/rpc/src/chain_api.rs

```diff
@@ -95,20 +94,14 @@ where
                 &start_ts,
                 recent_roots,
                 VoidAsyncWriter::default(),
-                compressed,
+                true,
```

Comment:
For the sake of readability, I'd avoid `true/false` in non-obvious method calls. When you read this code as someone who does not know the parameters to `export` by heart, do you know what `true` stands for here? Let's just put a `let compressed = true` somewhere. What do you think? Alternatively we could use more explicit enums, but it may be an overkill.

Reply:
Yep, `rust-analyzer` just make this code look like `compressed: true`

Reply:
I don't believe we should rely on the IDE features when it comes to readability. Browsing via GitHub should provide a similar experience (especially useful when reviewing PRs).